databases/week1/main.sql

```diff
@@ -0,0 +1,86 @@
+/*1*/
+SELECT
+SUM(1)
+FROM
+task;
+
+/*2*/
+SELECT
+SUM(1)
+FROM
+task
+WHERE
+due_date IS NULL;
+
+
+/*3*/
+SELECT  * 
+FROM
+ task 
+ JOIN status ON status.id= task.status_id
+ WHERE
+ status.name="Done";
+
+ /*4*/
+SELECT  * 
+FROM
+ task 
+ JOIN status ON status.id= task.status_id
+ WHERE
+ status.name!="Done";
+
+ /*5*/
+SELECT *
+FROM
+task
+ORDER BY
+updated DESC;
```

Comment:
We're looking at most recently created, so instead of updated you should use 'created'